tests/e2e-udx/udx-cpp/70-verify-cpp-source.yaml

```diff
@@ -22,7 +22,12 @@ data:
     set -o errexit
 
     POD_NAME=v-udx-cpp-sc1-0
-    UDX_OP=$(kubectl exec $POD_NAME -i -c server -- bash -c "cd /opt/vertica/sdk/examples && sed -i 's/python2/python/g' SourceFunctions.sql && vsql -U dbadmin -f SourceFunctions.sql 2>&1 > /tmp/SourceFunctionsOut.txt; cat /tmp/SourceFunctionsOut.txt")
+    container_count=$(kubectl get pod $POD_NAME -o jsonpath='{.spec.containers[*].name}' | wc -w)
+    echo "$container_count"
+    PYTHON_REG_EXP=python2
+    if [ "$container_count" -eq 2 ]; then PYTHON_REG_EXP=\$PYTHON3; fi
```

Comment:
Is there a PR on server side you are referring to? I am curious how the count of containers is associated with the python version?

Reply:
Counting the containers is just  an easier way to differentiate between udx-vcluster and udx-admintools. 2 containers(nma and server) means vcluster.

Reply:
Then, Do we keep python2 for admintools and python3 for vcluster?

Reply:
This will help you understand the issue https://git.verticacorp.com/projects/VER/repos/server/commits/d338d028bb47b4288d9fde11a8ccd546681aafc9#vertica/VerticaSDK/examples/SourceFunctions.sql